examples/execution_summary_example.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2015-2015 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from __future__ import print_function
+import os
+import shutil
+import time
+import datetime
+
+import luigi
+
+
+class MyExternal(luigi.ExternalTask):
+
+    def complete(self):
+        return False
+
+
+class Boom(luigi.Task):
+    task_namespace = 'examples'
```

Comment:
Is it any good practice to use namespaces? I was hoping that we get rid of them and use fqcn.